<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>30</y>
     <width>151</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>152</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ADC Options</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_adc_stop">
       <property name="text">
        <string>Stop Capture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_adc_static">
       <property name="text">
        <string>Static Freq</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_adc_dynamic">
       <property name="text">
        <string>Dynamic Freq</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Interv/hms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_adc_freq">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_adc_page">
       <property name="text">
        <string>ADC Page</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>30</y>
     <width>151</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Data Sending</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_UART">
       <property name="text">
        <string>UART</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_Ethernet">
       <property name="text">
        <string>Ethernet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_modbus">
       <property name="text">
        <string>Modbus(UART)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_mqtt">
       <property name="text">
        <string>MQTT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_start_sending">
       <property name="text">
        <string>Start Sending</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>30</y>
     <width>151</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>240</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Camera Options</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_camera_stop">
       <property name="text">
        <string>Stop Capture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_camera_static">
       <property name="text">
        <string>Static Freq </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_camera_dynamic">
       <property name="text">
        <string>Dynamic Freq</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Inter/hms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_camera_freq">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_camera_dynamic_monitor">
       <property name="text">
        <string>Dynam Monitor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_camera_page">
       <property name="text">
        <string>Camera Page</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_exit">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>233</y>
     <width>151</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connection: Offline</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
